src/data-tutorials/users.tsx

```diff
@@ -962,11 +962,18 @@ const Users: User[] = [
   },
   {
     title: 'How to Assemble a Web3 Dream Team and Build a High Value dApp',
-    description: 'There are multiple factors to consider when assembling a web3 dream team.',
```

Comment:
Hey you change someone else Description Here Revert it back to Original description